source/guide_baikal.rst

```diff
@@ -0,0 +1,96 @@
+.. highlight:: console
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: lang-php
+.. tag:: web
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/baikal.png
+      :align: center
+
+######
+Baïkal
+######
+
+.. tag_list::
+
+Baïkal_ is a lightweight CalDAV+CardDAV server.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`PHP <lang-php>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+We’re using :manual:`PHP <lang-php>` in the stable version 7.2:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.2'
+ [isabell@stardust ~]$
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+To install Baïkal we download the current version from the github release. ``cd`` to your :manual:`DocumentRoot <web-documentroot>` so the zip file will be under your ``html``.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ wget https://github.com/sabre-io/Baikal/releases/download/0.6.1/baikal-0.6.1.zip
+ [isabell@stardust html]$ unzip baikal-0.6.1.zip
+ [isabell@stardust html]$ rm baikal-0.6.1.zip
+ [isabell@stardust html]$ mv baikal/* ./
+ [isabell@stardust html]$ rmdir baikal
+ [isabell@stardust html]$ 
+
+Configuration
+=============
+
+After the installation you need to open isabell.uber.space/html in your browser to finish your setup.
```

Comment:
The [official guide](https://sabre.io/baikal/install/#apache-vhost-installation) suggests that only the `html` directory should be served. This also enables `/.well-known/carddav` and `/.well-known/caldav` via the included htaccess file. Do you mind moving the `baikal` directory to `/var/www/virtual/$USER/baikal`, symlinking `.../virtual/$USER/baikal` to `.../virtual/$USER/html`?